katana/utils/user_utils.py

```diff
@@ -3,6 +3,82 @@
 from django.conf import settings
 from utils.navigator_util import Navigator
 
+from .directory_traversal_utils import join_path, file_or_dir_exists
+
+DOTDATA = '.data'
+WAPPLOGS = 'wapp_logs'
+WAPPSDATA = 'wapps_data'
+USERDATA = 'user_data'
+
+
+class UserData:
+    # constructor for class User_data
+    def __init__(self, wapp_name, username=None):
+        self.username = username
```

Comment:
@bennfnc, it will be more foolproof if the username is not stored as an instance variable and the user is verified with each function call.